test/server/workers/testworkers.cpp

```diff
@@ -307,11 +290,23 @@ void TestWorkers::testConvertToPlaceholder() {
             ofs << "Some content.\n";
         }
 
-        exitCause = ExitCause::Unknown;
-        exitCode = _syncPal->_executorWorker->convertToPlaceholder(relativeFilePath, true, exitCause);
-        CPPUNIT_ASSERT_EQUAL(exitCode, ExitCode::Ok);
-        CPPUNIT_ASSERT_EQUAL(exitCause, ExitCause::Unknown);
+        exitInfo = _syncPal->_executorWorker->convertToPlaceholder(relativeFilePath, true);
+        CPPUNIT_ASSERT_EQUAL(ExitCode::Ok, exitInfo.code());
+        CPPUNIT_ASSERT_EQUAL(ExitCause::Unknown, exitInfo.cause());
     }
 }
 
+void TestWorkers::testHandleOpsExecutionError() {
+    SyncOpPtr opPtr = std::make_shared<SyncOperation>();
```

Comment:
What is tested by this test?

Reply:
Here is the test: [3298a55](https://github.com/Infomaniak/desktop-kDrive/pull/356/commits/3298a55252732c145ee6a75e97c6c16177c73bd2)

This was only for testing handleExecutorError behavior in debug mode and should not have been committed.